tests/basic/longevity_test.go

```diff
@@ -208,11 +222,11 @@ func testTrigger(wg *sync.WaitGroup,
 			}*/
 
 			triggerFunc(contexts, triggerEventsChan)
-			logrus.Infof("Trigger Function completed for [%s]\n", triggerType)
+			dash.Infof("Trigger Function completed for [%s]\n", triggerType)
 
 			//if isDisruptiveTrigger(triggerType) {
 			triggerLoc.Unlock()
-			logrus.Infof("Successfully released lock for trigger [%s]\n", triggerType)
+			dash.Infof("Successfully released lock for trigger [%s]\n", triggerType)
```

Comment:
this may not be required for dash, put messages which are critical to see the progress (Here and in other places)

Reply:
fixed